app/layout.tsx

```diff
@@ -16,7 +35,8 @@ export default function RootLayout({
 }) {
   return (
     <html lang="en">
-      <body className={inter.className}>{children}</body>
+      <Header />
+      <body className={`${abrilFatface.variable} ${pollerOne.variable} ${abhayaLibre.variable}`}>{children}</body>
     </html>
   )
```

Comment:
- [ ] Excellent job using `Header` component inside the layout so that every page have at least one header, However there is one major issue how you rendering your header. The header tag should be inside the body tag. You can render header tag out side the body tag it will affect your dom tree. To see the error details have a look
![Capture2](https://github.com/NurkaAmre/nomad/assets/35267447/9a33c237-45bd-4f05-a0a9-8fcb7250d0af)

***To Resolve the issue you have create a UI layout or you can render the `Header` component inside the body tag***